frontend/src/app/shared/pipes/cater-capacity.pipe.ts

```diff
@@ -0,0 +1,30 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { CanCaterFor } from 'digital-fuesim-manv-shared';
+
+export type CatererType = 'material' | 'personnel';
+
+@Pipe({
+    name: 'caterCapacity',
+})
+export class CaterCapacityPipe implements PipeTransform {
+    transform(canCaterFor: CanCaterFor, type: CatererType): string {
+        let result = '';
+        if (type === 'personnel') {
+            result += 'Diese Einsatzkraft ';
+        } else {
+            result += 'Dieses Material ';
+        }
+        if (canCaterFor.logicalOperator === 'or') {
+            result +=
+                'kann nur Patienten der selben Verletzungsschwere helfen.\n';
```

Comment:
```suggestion
                'kann nur Patienten einer Verletzungsschwere gleichzeitig helfen.\n';
```

Reply:
I'll update this, but we'll refactor this pipe to be a component as mentioned in Julians message above.